main.go

```diff
@@ -45,7 +45,9 @@ var (
 	disableSchedulerCollector        = flag.Bool("collector.scheduler.disable", false, "Disables the metrics collector for Cloud Scheduler")
 	disableStorageCollector          = flag.Bool("collector.storage.disable", false, "Disables the metrics collector for Cloud Storage")
 
-	enableExtendedMetricsGKECollector = flag.Bool("collector.gke.extendedMetrics.enable", false, "Enable the metrics collector for Google Kubernetes Engine (GKE) to collect ControlPlane and NodePool metrics")
+	enableExtendedMetricsGKECollector 		 							= flag.Bool("collector.gke.extendedMetrics.enable", false, "Enable the metrics collector for Google Kubernetes Engine (GKE) to collect ControlPlane and NodePool metrics")
```

Comment:
Please run the following before submitting PRs; I will add these to the GitHub Actions workflow

1. `go fmt ./...`
1. `golangci-lint run ./...`

There's a v2 for `golangci-lint` (`go install github.com/golangci/golangci-lint/v2/cmd/golangci-lint@latest`)

Reply:
`ControlPlane` isn't a thing like `NodePool`, I think it should say `control plane or NodePool metrics`